test/replay/_replay.py

```diff
@@ -0,0 +1,94 @@
+import sys, os
+import time, datetime
+scriptDir= os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(scriptDir, ".."))
+import argparse
+from stresstest import stressClient
+import csv
+from pickle import FALSE,TRUE
+import string
+
+def workaround_dict(a,b,timeout=0):
+    pass
+
+def getTimeDiff(timestamps):
+    times = []
+    time = []
+    hours = []
+    minutes = []
+    seconds = []
+    microseconds = []
+    timeDiff = [0]
+
+    for i in range(0,len(timestamps)):
+        times.append(timestamps[i][timestamps[i].find(" ")+1:])     #take only the information about time from the timestamp
+        
+        #cut out corresponding information from the time string
+        hours.append(int(times[i][0:2]))
+        minutes.append(int(times[i][3:5]))
+        seconds.append(int(times[i][6:8]))
+        microseconds.append(int(times[i][9:]))
+
+        #calculate time delta in seconds by adding up all the values with their correlating factor (i.e. 1h = 3600s)
+        time.append((float(microseconds[i]*10**-6)+float(seconds[i])+float(minutes[i]*60)+float(hours[i]*3600)))
```

Comment:
`timedelta` https://www.geeksforgeeks.org/python-datetime-timedelta-function/ does not work?

Reply:
You would have to make the same calculations as in my solution, so I assumed it to be unnecessarily complicated

Reply:
I gave you a suggestion for the calculation in your new code